idaes/power_generation/carbon_capture/mea_solvent_system/property_package/liquid_prop.py

```diff
@@ -0,0 +1,1282 @@
+
+        # Liquid properties and state block
+        m.fs.properties = LiquidParameterBlock()
+        m.fs.state_block = m.fs.properties.state_block_class(
+            default={"parameters": m.fs.properties})
+
+        #fix state variables
+        m.fs.state_block.flow_mol.fix(22)
+        m.fs.state_block.temperature.fix(383)
+        m.fs.state_block.pressure.fix(101325)
+        m.fs.state_block.mole_frac["CO2"].fix(0.05)
+        m.fs.state_block.mole_frac["H2O"].fix(0.80)
+        m.fs.state_block.mole_frac["MEA"].fix(0.15)
+
+
+        #add properties
+        hasattr(m.fs.state_block, "flow_mol_comp")
+        hasattr(m.fs.state_block, "mw_ave")
+        hasattr(m.fs.state_block, "mass_frac")
+        hasattr(m.fs.state_block, "mass_frac1")
+        hasattr(m.fs.state_block, "vol_mol")
+        hasattr(m.fs.state_block, "vol_mol_comp")
+        hasattr(m.fs.state_block, "conc")
+        hasattr(m.fs.state_block, "conc_comp")
+        hasattr(m.fs.state_block, "dens_mass")
+        hasattr(m.fs.state_block, "k_eq_conc")
+        hasattr(m.fs.state_block, "conc_comp_true")
+        hasattr(m.fs.state_block, "cp_mol")
+        hasattr(m.fs.state_block, "cp_mol_mean")
+        hasattr(m.fs.state_block, "cp_mol_comp")
+        hasattr(m.fs.state_block, "cp_mol_comp_mean")
+        hasattr(m.fs.state_block, "diffus")
+        hasattr(m.fs.state_block, "cp_mass_comp")
+        hasattr(m.fs.state_block, "cp_mass_comp_mean")
+        hasattr(m.fs.state_block, "enth_mol_mean")
+        hasattr(m.fs.state_block, "visc_d")
+        hasattr(m.fs.state_block, "thermal_cond")
+        hasattr(m.fs.state_block, "surf_tens")
+        hasattr(m.fs.state_block, "pressure_sat")
+        hasattr(m.fs.state_block, "k2_rxn")
+        hasattr(m.fs.state_block, "habs")
+        hasattr(m.fs.state_block, "hvap")
+        hasattr(m.fs.state_block, "act_coeff_NRTL")
+        hasattr(m.fs.state_block, "henry_N2O_analogy")
+        hasattr(m.fs.state_block, "henry_NRTL")
+
+        #initialize variables
+        m.fs.state_block.initialize()
+        report_statistics(m.fs)
+
+        print('degrees of freedom = {}\n'.format(degrees_of_freedom(m)))
+
+        print('============Variables======================================')
+        for k in m.fs.state_block.component_objects(Var,descend_into =True):
+            print('\n{}'.format(k.parent_component().doc))
+            for i,j in k.iteritems():
+                print('{} = {:0.4g}'.format(
+                               str(j).replace('fs.state_block.',''),value(j)))
+
+        print('\n============Expressions=====================================')
+        for k in m.fs.state_block.component_objects(
+            Expression,descend_into =True):
+            print('\n{}'.format(k.parent_component().doc))
+            for i,j in k.iteritems():
+                print('{} = {:0.4g}'.format(
+                             str(j).replace('fs.state_block.',''),value(j)))
+
+if __name__ == '__main__':
```

Comment:
This can be removed - no-one should be trying to "run" a property package.

Reply:
Done. code removed